smooth/framework/functions/print_results.py

```diff
@@ -7,20 +7,33 @@ def print_smooth_results(smooth_results):
 
     # Calculate the sum of all total annuities [EUR/a].
     sum_of_tot_annuity = 0
+    sum_of_tot_emissions_annuity = 0
```

Comment:
maybe rename to sum_of_tot_ann_emissions as annual emissions

Reply:
I wanted to be precise on the fact that is sums up the total emissions, in which total emissions again is the sum of fixed and variable emissions.

Reply:
okay, i think we mean the same but we're talking at cross purposes :-D 
my proposal also includes the sum_of... but without the word annuity in it

Reply:
https://github.com/rl-institut/smooth/pull/15/commits/0dbf3c450fdd0c83f095bc27c95ec1e1b42bfcee